script/Ciel/Day 5/05_01_CIEL05_4.txt

```diff
@@ -911,23 +915,23 @@ That is the vampire who bears the name Vlov.
 [sha:40271ddac66e05b513cee6d2d7bb2555e0dc3a16]{
 -- Page 726, Offset 12818.
 -- その本質は寒さと裏切り、漂流と悔恨。
-His very essence consists of coldness and treason, of forsakenness and regret.
+His essence is cold and betrayal; drifting and remorse.
 }
 [sha:4e1f59e8f0679abbbaa153fc6b776840614c107c]{
 -- Page 727, Offset 12819.
 -- ああ―――これなら殺せる。
-%{i}Ahhh%{/i}―――With this, I can kill him.
+%{i}Indeed%{/i}, this one I can kill.
```

Comment:
Indeed feels way too posh here imo. _Ahh_ of satisfied understanding more natural to me.